Wasm/Engine.lean

```diff
@@ -83,25 +85,33 @@ def mkStore (m : Module) : Store m :=
   Store.mk $ instantiateFs m
 
 def funcByName (s : Store m) (x : String) : Option $ FunctionInstance m :=
-  match s.func.filter (fun f => f.name == .some x) with
```

Comment:
Note to self: are names now called labels (or what's the name for those dollar sign thingies?)

Reply:
I think they are called 'symbolic identifiers' in the spec, but yeah, good question, it's confusing me as well. I think labels are something completely different though.

Reply:
Let's keep this comment thread as a nagging reminder to make an issue?

Reply:
Labels are another type of [stack values](https://webassembly.github.io/spec/core/exec/runtime.html#labels) and are necessary for [proper branching](https://webassembly.github.io/spec/core/syntax/instructions.html#control-instructions)!

Reply:
Thanks for digging it up!